packages/compiler-sfc/src/script/importUsageCheck.ts

```diff
@@ -20,7 +20,7 @@ export function isImportUsed(local: string, sfc: SFCDescriptor): boolean {
   return new RegExp(
     // #4274 escape $ since it's a special char in regex
     // (and is the only regex special char that is valid in identifiers)
-    `[^\\w$_]${local.replace(/\$/g, '\\$')}[^\\w$_]`
+    `[^\\w$.]${local.replace(/\$/g, '\\$')}[^\\w$]`
```

Comment:
Don't we also need to account for `.` being used as part of a spread operator? e.g.:

https://deploy-preview-8897--vue-sfc-playground.netlify.app/#__DEV__eNp9UT1PwzAQ/SsnL10qd+hWpZUAFQkGQMDoJU2vwcXxWbZTIiL/d86OWhhQt9P7sN/dG8WNc/LUo1iJKjReuwgBY+/A1LZdKxGDEhtldefIRxjhnggSHDx1MJOLA5E8hpmy1WIys7SK2DlTR+QZoNrr02YcYZRSsjdBStUiY9lzUYo5f9SQPeiW3yPLacbsVqKhzmmD/tlFTZbDrKAwmauNoa/HgkXf4/yMNx/YfP6DH8OQMSVePAb0J1TiwsXatxgnevv2hAPPF7KjfW9YfYV8xUCmzxkn2W1v9xz7j66kfSh31LZ9D9shog3npXLQrExFrwRXcndl9d+4S7ksPmUTX3EqhM+HQymMbxpiKW09mXe1X8FsV39zaewR6QelJakA

I also wonder whether we should account whitespace after `.` operators. e.g. `Foo . Bar . Baz`?